ts/src/transports/fetch.ts

```diff
@@ -40,7 +41,8 @@ class Fetch implements Transport {
         detach(() => {
           this.options.onChunk(result.value);
         });
-        return this.pump(this.reader, res);
+        this.pump(this.reader, res);
+        return;
```

Comment:
I think this return statement is redundant and can be removed

Reply:
You're right, have removed.

Reply:
Removing this caused CI to fail - 'Not all code paths return a value'

Reply:
Oh dear that's 2/2 for bad suggestions from me 😂 Apologies for any time wasted and thanks again for the patch